src/rdkafka_conf.c

```diff
@@ -3709,12 +3709,6 @@ const char *rd_kafka_conf_finalize(rd_kafka_type_t cltype,
                                        "is mandatory when "
                                        "`sasl.oauthbearer.method=oidc` is set";
                         }
-
-                        if (!conf->sasl.oauthbearer.extensions_str) {
```

Comment:
@kirktrue @jliunyu Are extensions indeed optional for OIDC?

Reply:
My reading of https://datatracker.ietf.org/doc/html/rfc7628#section-3.1 is that they are optional (emphasis is my own):

> Client responses are a GS2 [RFC5801] header followed by **zero or more** key/value pairs, or it **may be empty**.